test/integration/forms/action.js

```diff
@@ -672,6 +674,7 @@ context('Patient Action Form', function() {
 
   specify('submitting the form', function() {
     cy
+      .routesForPatientAction()
       .routeAction(fx => {
```

Comment:
It shouldn't cause any issues if `.routeAction()` is added twice here, right? Since `.routesForPatientAction()` includes `.routeAction()` as well.

Running the test locally, it doesn't appear to cause any issues.

Reply:
yeah the 2nd will overwrite it.. for this reason I contemplated if `routesForPatientAction` shouldn't include `routeAction` at all or if I should add all of the rest of the routes from that command here.. but the simplest was just to duplicate it..  I _think_ that's reasonable?  thoughts?

Reply:
That seems reasonable to me. Similar scenarios will probably happen when using these commands in the future. So just wanted to make sure.